### DISCUSSION TITLE
In case you have been wondering how OCP (and OKD) images are build by RH CI infrastructure

### DISCUSSION THREAD

**Discussion by fortinj66** [okd-project/okd#1284] [General]
https://docs.ci.openshift.org/docs/architecture/images/

This is specifically for OCP but I would figure OKD is pretty similar and the image definitions are stored in a similar spot.

I found this very interesting as I wondered where the list of objects/images used in OCP/OKD was managed...  This could possible lead to some type of external process to build OKD outside of the RH CI infrastructure

**Comment by vrutkovs** [comment]
>This could possible lead to some type of external process to build OKD outside of the RH CI infrastructure

Yes, that's the plan. Folks from operate-first.cloud have the infra which could be used to build images and prepare test payloads to experiment with OKD

**Reply by fortinj66** [reply]
I've gotten a base PROW instance running on my local cluster connected to github.  I'm going to play and see if I can get some of the OCP/OKD jobs to run on it
